File: examples/close-session/rest/python/close_session.py
# Creates a Browserless session and closes it via the Session API.
#
# Install: pip install requests
# Run:     python close_session.py

import requests

TOKEN = "YOUR_API_TOKEN_HERE"

session_response = requests.post(
    f"https://production-sfo.browserless.io/session?token={TOKEN}",
    json={"ttl": 60000, "stealth": True},
)
session = session_response.json()
print("Session created:", session["id"])

close_response = requests.delete(f"{session['stop']}&force=true")
print("Session closed:", "success" if close_response.ok else "failed")

File: examples/close-session/rest/nodejs/close-session.mjs
// Creates a Browserless session and closes it via the Session API.
//
// Run: node close-session.mjs

const TOKEN = "YOUR_API_TOKEN_HERE";

const sessionResponse = await fetch(
  `https://production-sfo.browserless.io/session?token=${TOKEN}`,
  {
    method: "POST",
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify({ ttl: 60000, stealth: true }),
  }
);
const session = await sessionResponse.json();
console.log("Session created:", session.id);

const closeResponse = await fetch(`${session.stop}&force=true`, {
  method: "DELETE",
});
console.log("Session closed:", closeResponse.status === 200 ? "success" : "failed");
